manticore/manticore.py

```diff
@@ -37,12 +37,15 @@ def makeDecree(args):
     platform.input.transmit(initial_state.symbolicate_buffer('+'*14, label='RECEIVE'))
     return initial_state
 
-def makeLinux(program, argv, env, concrete_start = ''):
+def makeLinux(program, argv, env, concrete_start = '', symbolic_files=None):
```

Comment:
since makeLinux is only used once, and always called with symbolic files argument, we can keep it simple and not make this a default arg?